New_Website/config/01-base.raku

```diff
@@ -5,25 +5,44 @@
                             #| for each source file that has been rendered. Intended to avoid
                             #| re-rendering every source when only some have been modified
     :canonical<en>,         #| sub-dir of sources with canonical content
-    :L10N<L10N>,            #| directory with translation information
+    :misc<misc>,            #| directory with translation & other information
     :ui-dictionary<ui-dictionary.rakuon>, #| name of dictionary with ui token and translations
     :extensions<rakudoc rakumod>, #| possible extension of rakudoc source
     :!quiet,                #| no output is required if True
-    :with-only(),           #| only render files in this list
+    :with-only<
+        Language/operators Language/REPL Language/variables Language/math Language/101-basics Type/AST
+        Type/Anytype/IO/Handle Type/IO/CatHandle Type/IO/Special Type/IO/ArgFiles Type/IO/Socket
+        Type/IO/Notification Type/IO/Pipe Type/IO/Spec Type/IO/Pathtype/RakuAST/Doc
+        RakuDoc_v2
+        about 403 404 license
+    >,           #| only render files in this list
```

Comment:
How big should we expect this with-only list to get to handle all of Raku's docs?

Reply:
I see, if it's _empty_ everything is rendered. Got it.